src/unix/udp.c

```diff
@@ -32,6 +32,10 @@
 #endif
 #include <sys/un.h>
 
+#if defined(__linux__) && defined(MSG_ERRQUEUE)
```

Comment:
Here and below:
```suggestion
#if defined(__linux__)
```
Not necessary, MSG_ERRQUEUE is always defined. (It's been around since at least linux 2.2.x)

Reply:
Done !